src/go-postgres/devcontainer-template.json

```diff
@@ -7,19 +7,18 @@
     "publisher": "Dev Container Spec Maintainers",
     "licenseURL": "https://github.com/devcontainers/templates/blob/main/LICENSE",
     "options": {
-		"imageVariant": {
+        "imageVariant": {
             "type": "string",
             "description": "Go version (use -bookworm, or -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
                 "1-bookworm",
+                "1.21-bookworm",
                 "1.20-bookworm",
                 "1.19-bookworm",
                 "1-bullseye",
+                "1.21-bullseye",
                 "1.20-bullseye",
-                "1.19-bullseye",
-                "1-buster",
-                "1.20-buster",
-                "1.19-buster"
+                "1.19-bullseye"
```

Comment:
1.19 has reached EOL and is deprecated. See https://github.com/docker-library/golang/pull/479